server/form-pages/apply/risks-and-needs/risk-of-serious-harm/summary.ts

```diff
@@ -150,40 +150,57 @@ export default class Summary implements TaskListPage {
               text: 'Children',
             },
             {
-              text: riskData.value.riskToChildren,
-              classes: `rosh-widget__risk--${riskData.value.riskToChildren.toLowerCase().replace(/ /g, '-')}`,
+              text: source.riskToChildren || 'No Data',
+              classes: `rosh-widget__risk--${source.riskToChildren?.toLowerCase().replace(/ /g, '-')}`,
             },
           ],
           [
             {
               text: 'Public',
             },
             {
-              text: riskData.value.riskToPublic,
-              classes: `rosh-widget__risk--${riskData.value.riskToPublic.toLowerCase().replace(/ /g, '-')}`,
+              text: source.riskToPublic || 'No Data',
+              classes: `rosh-widget__risk--${source.riskToPublic?.toLowerCase().replace(/ /g, '-')}`,
             },
           ],
           [
             {
               text: 'Known adult',
             },
             {
-              text: riskData.value.riskToKnownAdult,
-              classes: `rosh-widget__risk--${riskData.value.riskToKnownAdult.toLowerCase().replace(/ /g, '-')}`,
+              text: source.riskToKnownAdult || 'No Data',
+              classes: `rosh-widget__risk--${source.riskToKnownAdult?.toLowerCase().replace(/ /g, '-')}`,
             },
           ],
           [
             {
               text: 'Staff',
             },
             {
-              text: riskData.value.riskToStaff,
-              classes: `rosh-widget__risk--${riskData.value.riskToStaff.toLowerCase().replace(/ /g, '-')}`,
+              text: source.riskToStaff || 'No Data',
+              classes: `rosh-widget__risk--${source.riskToStaff?.toLowerCase().replace(/ /g, '-')}`,
             },
```

Comment:
this could be abstracted out into a new method to make it easier to read but I'll do that later once we have this feature deployed